Vladyslav_Gurevych/HW5/Task3.py

```diff
@@ -0,0 +1,19 @@
+# there is a string " name=Amanda=sssss&age=32&&salary=1500&currency=euro ".
+# Convert this string to a dictionary {name: Amanda, age: 32, salary: 1500, currency: quro}
+
+staff = " name=Amanda=sssss&age=32&&salary=1500&currency=euro "
+staff_new = staff.lstrip().rstrip().replace("&&","&").split("&")
+staff_new = " ".join(staff_new).replace("=sssss", "")
+staff_new = "".join(staff_new).replace("=", " ").split(" ")
```

Comment:
why do you replace '=' with ' ' and then split by ' ' .
You can split by '=' without replace